baybe/parameters/numerical.py

```diff
@@ -27,6 +27,7 @@ class NumericalDiscreteParameter(DiscreteParameter):
     # object variables
     # NOTE: The parameter values are assumed to be sorted by the tolerance validator.
     _values: tuple[float, ...] = field(
+        alias="values",
```

Comment:
Is this related to the changes made here resp. necessary or just added since this was forgotten previously?